tests/simple_remsql.test/runit

```diff
@@ -27,6 +27,41 @@ for file in $files ; do
 done
 
 
+if [[ $DBNAME == *"whitelistgenerated"* ]] ; then
```

Comment:
test output and error code when access is denied

Reply:
Added testcase for access denied. Returning correct error msg for access error.